uQR.py

```diff
@@ -1284,3 +1286,10 @@ def get_matrix(self):
         code += [[False]*width] * self.border
 
         return code
+
+    def render_matrix(self):
+        out = ""
+        for row in self.get_matrix():
+            out += "".join([{False: " ", True: "█"}[x] if x in (False, True) else "╳" for x in row])
+            out += "\n"
+        return out
```

Comment:
Although it's not in keeping with the current quality of the library, if exposing a new public method I'd like to put it under test and documentation so that we can make sure it works in the future. :)

It looks like the intent of this method is to create a string representation of the matrix that can be viewed in the console:
  * I tend to think more about writing QR codes to displays, for which the True/False representation is sufficient.
  * But also I can see how this would be useful to people working at the REPL, especially for debugging purposes.

If you'd like to keep it, could you add test and documentation?

One more questions: does this approach cause a problem in terms of aspect ratio? I'm thinking that each of those characters is 8x16. Does this lead to an unusably distorted code. That does have to be a problem if you've already used this code and it's useful to you, though.

Reply:
The 8x16 aspect ratio is legit for squares - you can see an example in the readme at https://github.com/i-infra/uqr (wasn't sure if this PR would go anywhere, but glad I tried!)

Reply:
I can add a comment and test no problemo. Although I'm not sure you're presently testing with ie) the micropython unix port? I have a binary built from Pycopy v3.1.5 that I did my dev with...